internal/jobcontainers/jobcontainer.go

```diff
@@ -401,7 +401,7 @@ func (c *JobContainer) waitBackground(ctx context.Context) {
 	// them to exit.
 	<-c.init.proc.waitBlock
 
-	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
+	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
```

Comment:
why?

Reply:
Felt 10 might've been a bit too long but I'm fine with keeping it

Reply:
I might've just been trying out different values and never reverted but that was my original train of thought from trying to remember